src/ecc/hash.js

```diff
@@ -32,7 +32,11 @@ function HmacSHA256(buffer, secret) {
 }
 
 function ripemd160(data) {
-    return createHash('rmd160').update(data).digest()
+    try{
```

Comment:
could you add space after `try` and `catch(e)`

Reply:
hey, let's approve and I will clean up on the other side.  I need to add release notes and version it.

Reply:
Ok .. I would approve but it still says "Merging is blocked" ..